public/locales/en-US/send.ftl

```diff
@@ -195,13 +196,13 @@ fileCount = { $num ->
     [one] 1 file
    *[other] { $num } files
 }
+# $size refers to megabytes or gigabytes
```

Comment:
My question is more about: how do you format the number? Can you point me to the code?

I have the feeling it's old code, but it would be nice to double check. Different locales will have different formatting options, potentially names for units (it can also be a follow-up if needed, to speed things up)

Reply:
Yep: https://github.com/mozilla/send/blob/master/app/utils.js#L98

Reply:
I'll carry this over as a comment on danny's PR so we can merge this branch into his